contracts/PoolInfo.vy

```diff
@@ -99,4 +106,37 @@ def get_pool_info(_pool: address) -> PoolInfo:
         rates: Registry(registry).get_rates(_pool),
         lp_token: Registry(registry).get_lp_token(_pool),
         params: Registry(registry).get_parameters(_pool),
+        is_meta: Registry(registry).is_meta(_pool),
+        name: Registry(registry).get_pool_name(_pool),
     })
+
+
+@external
+def set_pool_asset_type(_pool: address, _asset_type: String[32]):
```

Comment:
I think at some point I was debating two alternatives in terms of tracking asset type, and eventually settled on just cache-ing the data on-chain via the mapping + getter + setter. I think my train of thought was this would be used primarily off-chain so would be a good fit in the PoolInfo contract.
Plus I'm not sure if we want to standardize the asset type logic? In commit 5aa6346 I've moved the functionality over to the registry, which thinking about it makes more sense, since it keeps pool information consolidated in the registry.